kwok/charts/crds/karpenter.sh_nodeclaims.yaml

```diff
@@ -32,6 +32,9 @@ spec:
         - jsonPath: .status.conditions[?(@.type=="Ready")].status
           name: Ready
           type: string
+        - jsonPath: .status.conditions[?(@.type=="Drifted")].status
```

Comment:
How common do you think it will be for folks to look this up? I definitely can see it as useful -- what do you think about starting this in the `-o wide` output with `priority=1` and then elevating it if we hear from more folks that they'd like to see it in the non-wide output?

Reply:
Sounds good to me.